scripts/gha/desktop_tester.py

```diff
@@ -55,7 +59,7 @@ def main(argv):
   testapps = []
   for file_dir, _, file_names in os.walk(testapp_dir):
     for file_name in file_names:
-      if file_name == testapp_name:
+      if file_name == testapp_name and "ios_build" not in file_dir:
```

Comment:
Why are we excluding this one particular directory?  What about excluding android, for instance?  Its worth a comment.

Reply:
Added.